java/java-ranger-regression/schedule_eqchk/Schedule2.java

```diff
@@ -0,0 +1,321 @@
+
+public class Schedule2 {
+	//
+	public static final int NEW_JOB = 1;
+	public static final int UPGRADE_PRIO = 2;
+	public static final int BLOCK = 3; 
+	public static final int UNBLOCK = 4;
+	public static final int QUANTUM_EXPIRE = 5;
+	public static final int FINISH = 6;
+	public static final int FLUSH = 7;
+	
+	//
+	public static final int MAXPRIO = 3;
+	//
+	public static final int MAXLOPRIO = 2;
+	//
+	public static int next_pid = 0;
+	
+	//
+	public static Queue current_job; 
+	//
+	public static Queue[] prio_queue; 
+	
+	public static Process[] block_queue;
+	public static final int block_length = 3;
+	public static int block_in;
+	public static int block_out;
+	
+	public static void main(String[] args) {
```

Comment:
nit-pick: I think this main method is not used and should be removed. So far, I haven't found a definition regarding the amount of allowed main methods in the code, but removing it might clean the code.

Reply:
Done. Thanks for catching this!